### PR TITLE
hotfix showOnLedger registering TmFormMsg (should work)

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
OK, now is good. I had to start it from the `master` branch. We really need to polish these differences between the two branches

**Comment by Bitcoinera** [comment]
Let's merge & hotfix! :muscle:

File: app/src/components/account/UserMenuAddress.vue [hunk 1]
```diff
@@ -30,12 +30,14 @@
 </template>
 <script>
 import Address from "common/Address"
+import TmFormMsg from "src/components/common/TmFormMsg"
```
Comment (jbibla) [inline app/src/components/account/UserMenuAddress.vue:33]: what does this look like? can you share a screenshot?

Comment (Bitcoinera) [reply app/src/components/account/UserMenuAddress.vue:33]: :sweat_smile: just by looking at the code you the component is not registered, don't you?

I'm surprised this is taking so long to be merged, such a simple bug :shrug:

Comment (Bitcoinera) [reply app/src/components/account/UserMenuAddress.vue:33]: image

**Comment by Bitcoinera** [comment]
I recommend merging this to master, since this functionality is now broken on production

File: app/src/components/account/UserMenuAddress.vue (hunk 1 above)
Comment (jbibla) [reply app/src/components/account/UserMenuAddress.vue:33]: i get it - i'm just surprised we're using this component for this - seems totally unnecessary.

**Pull request merged**